client/draft_README.md

```diff
@@ -0,0 +1,427 @@
+
+</div>
+</details>
+
+---
+
+## Contributors
+
+  <!-- TODO: [ ] - Requesting Section Review, Remove, Update, Addition, and or Completion By Abel, Cynthia, && Sergio-->
+
+Gratitude to the following people who have contributed to this project:
+
+<h3>Abel Belay</h3>
+<a href="mailto:[email]@[provider]" target="_blank">
+    <img src="https://img.shields.io/badge/-[email_address]-c14438?style=flat&logo=Gmail&logoColor=white">
+ </a>
+<div style='display: flex; justify-content: center; gap: .5rem'>
+
+<a href="https://www.linkedin.com/in/username" target="_blank">
+      <img src="https://img.shields.io/badge/Abel Belay-blue?style=flat&logo=Linkedin&logoColor=f32dc8">
+ </a>
+
+[@abel-belay](https://github.com/abel-belay)
+
+<a href="https://[username]g.github.io/portfolio" target="_blank">
+    <img src="https://img.shields.io/badge/[username]github.io/portfolio-gold?style=flat&logo=github"
+
+</a>
+
+</div>
+<h3>Cynthia Eddy</h3>
+<a href="mailto:[email]@[provider]" target="_blank">
+    <img src="https://img.shields.io/badge/-[email_address]-c14438?style=flat&logo=Gmail&logoColor=white">
+ </a>
+<div style='display: flex; justify-content: center; gap: .5rem'>
+
+<a href="https://www.linkedin.com/in/username" target="_blank">
+      <img src="https://img.shields.io/badge/Cynthia Eddy-blue?style=flat&logo=Linkedin&logoColor=f32dc8">
+ </a>
+
+[@cynthiaeddy](https://github.com/cynthiaeddy)
+
+<a href="https://[username]g.github.io/portfolio" target="_blank">
+    <img src="https://img.shields.io/badge/[username]github.io/portfolio-gold?style=flat&logo=github"
+
+</a>
+
+</div>
+
+<h3>Kathy Lambert</h3>
+<a href="mailto:Kathy@KathyLambert.me" target="_blank">
+    <img src="https://img.shields.io/badge/-Kathy@KathyLambert.me-c14438?style=flat&logo=Gmail&logoColor=white">
+ </a>
+
+<div style='display: flex; justify-content: center; gap: .5rem'>
+
+<a href="https://www.linkedin.com/in/k-lambert" target="_blank">
+      <img src="https://img.shields.io/badge/Kathy Lambert-blue?style=flat&logo=Linkedin&logoColor=f32dc8">
+ </a>
+
+[@CodeMeKathy](https://github.com/CodeMeKathy)
+
+ <a href="https://kathylambert.me" target="_blank">
+    <img src="https://img.shields.io/badge/kathylambert.me-gold?style=flat&logo=github"
+
+</a>
+
+</div>
+
+<h3>Njteh Keledjian</h3>
+<a href="mailto:[email]@[provider]" target="_blank">
+    <img src="https://img.shields.io/badge/-[email_address]-c14438?style=flat&logo=Gmail&logoColor=white">
+ </a>
+<div style='display: flex; justify-content: center; gap: .5rem'>
+
+<a href="https://www.linkedin.com/in/username" target="_blank">
+      <img src="https://img.shields.io/badge/Njteh Keledjian-blue?style=flat&logo=Linkedin&logoColor=f32dc8">
+ </a>
+
+[@njteh7](https://github.com/njteh7)
+
+<a href="https://[username]g.github.io/portfolio" target="_blank">
+    <img src="https://img.shields.io/badge/[username]github.io/portfolio-gold?style=flat&logo=github"
+
+</a>
+
+</div>
+
+<h3>Sergio Silvay</h3>
```

Comment:
just need to delete ending "y". should be : Sergio Silva

Reply:
Thank you for catching this typo @sergiobtos! I have made the correction.  Can I expect the rest of your changes on a sparate branch per the requested guidance? Please note, if you pefer not to include any of you social or contact information and it can be ommitted.  Thank you again!